e2e/tests/unfinished-polygons.test.js

```diff
@@ -103,9 +103,12 @@ Scenario("Saving polygon drawing steps to history", async function({ I, LabelStu
   assert.strictEqual(result[0].value.closed, true);
 
   I.say(`try to undo closing and 2 last points`);
-  I.pressKey(['CommandOrControl', 'Z']);
-  I.pressKey(['CommandOrControl', 'Z']);
-  I.pressKey(['CommandOrControl', 'Z']);
+  I.click("button[aria-label=Undo]");
+  I.click("button[aria-label=Undo]");
+  I.click("button[aria-label=Undo]");
+  // I.pressKey(['CommandOrControl', 'Z']);
+  // I.pressKey(['CommandOrControl', 'Z']);
+  // I.pressKey(['CommandOrControl', 'Z']);

```

Comment:
Are these meant to stay or be replaced strictly by the button usage?

Maybe can we instead split to another test to ensure both work correctly in the same fashion?

Reply:
they should be removed, as currently the hotkey undo is not working the same way as clicking the undo button and it should be addressed in another ticket